.github/workflows/PR_reviews.yml

```diff
@@ -10,8 +10,23 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         with:
-          msg: "Reviewers, please ensure that you check the below items. \n
-                [ ] Condition 1 \n
-                [ ] Condition 2 \n
-                [ ] Condition 3 \n"
-          check_for_duplicate_msg: true  # this is only commented once
\ No newline at end of file
+          msg: "Before a PR is accepted, it must meet the following criteria: \n
+                [ ] Is the necessary information provided? \n
+                  [ ] Does the PR have a complete description? Does it explain what the PR is attempting to do/fix, does it explain how it does this? \n
+                  [ ] Is there an explanation of why this PR is needed? \n
+                  [ ] Please use the following PR template: https://github.com/tardis-sn/tardis/blob/master/.github/PULL_REQUEST_TEMPLATE.md \n
+                [ ] Is this a duplicate PR? \n
+                  [ ] If a new PR is clearly a duplicate, ask how this PR is different from the original PR? \n
+                  [ ] If this PR is about to be merged, close the original PR with a link to this new PR that solved the issue. \n
+                [ ] Does it pass existing tests and are new tests provided if required? \n
+                  [ ] The test coverage should not decrease, and for new features should be at or very close to 100%. \n
+                [ ] Is the code properly documented? \n
+                  [ ] If this modifies existing code, then the docs should be updated. If this adds a new feature, additional documentation should be created. \n
+                  [ ] Sphinx and docstrings in the code (in numpydoc format) \n
+                [ ] Does this conform to PEP 8 and the TARDIS style guidelines? \n
```

Comment:
Where does "TARDIS style guidelines" live - can we add a link to them here?

Reply:
I don't think that we have these yet — currently an issue (#1170 ) and a PR (#1183 ). Would you recommend removing this line for now, waiting until we have a style guide to push this, or something else?

Reply:
Yeah I think we can remove TARDIS style guidelines and perhaps put `black` code style (with black hyperlinked).